common/src/main/java/com/wynntils/models/items/annotators/gui/AbilityTreeAnnotator.java

```diff
@@ -14,17 +14,32 @@
 import net.minecraft.world.item.ItemStack;
 
 public final class AbilityTreeAnnotator implements ItemAnnotator {
+    // Deals with the ability tree button in the compass menu
     private static final StyledText ABILITY_TREE_NAME = StyledText.fromString("§b§lAbility Tree");
     private static final Pattern ABILITY_POINTS_PATTERN = Pattern.compile("^§3✦ Unused Points: §f(\\d+)$");
 
+    // Deals with the reset button in the ability tree screen
+    private static final StyledText INTERNAL_ABILITY_TREE_NAME = StyledText.fromString("§3§lAbility Points");
```

Comment:
I dont understand what is happening here..

Reply:
There are two screens where the points are presented, the compass menu and the ability tree menu? They have different names on the two items that has the number of ability points in the lore. I don't think that is strange? 

Possibly the names could be made clearer: COMPASS_MENU_ABILITY_POINTS_NAME and ABILITY_TREE_ABILITY_POINTS_NAME or something like that.